src/app/testPage/page.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import SiteButton from "@/components/siteButton";
+import { useState } from "react";
+
+export default function TestPage() {
+  const [testEmailSent, setTestEmailSent] = useState(false);
+
+  //we need to be able to differentiate between what kind of email needs to be sent as a response to which action.
+  //I wonder if we need another folder in the routes section to divide the different API endpoints for the different types of emails?
+
+  const handleClick = async () => {
```

Comment:
You should be more descriptive of your click functions. Instead of generically calling them "handleClick" name them according to what they do -> "sendTestEmail"